### ISSUE TITLE
Failed to recreate to the paper figures

### ISSUE THREAD

**Issue by chedlyba** [Laborieux-Axel/SynapticMetaplasticityBNN#3]
I've been trying to recreate the results of the paper (figure 2-a) in order to understand the parameters of the model. However, I fail to have the desired effect using the commands proposed in the home page.
I get a chaotic learning curve with regards to task 1 and task 2 with a lr of 5e-3. With an lr of 1-e3, the model converges in each task (this time 4 tasks of pMNIST) but I still get catastrophic forgetting. 

![meta_1 35_lr_1e-3pMNIST_pMNIST_pMNIST_pMNIST](https://user-images.githubusercontent.com/84285750/183660249-bd8d03ef-fdbe-461e-9912-4f721a55ab20.png)
Fig 1: meta = 1.35, lr = 1e-3, task_sequence = pMNIST_pMNIST_pMNIST_pMNIST

![meta_1 35_lr_5e-3_pMNIST_pMNIST_pMNIST_pMNIST_pMNIST_pMNIST](https://user-images.githubusercontent.com/84285750/183660261-2b8ef6b1-f277-4cba-bb7d-ea0b4615c4dc.png)
Fig 2: meta = 1.35, lr = 5e-3, task_sequence = pMNIST_pMNIST_pMNIST_pMNIST pMNIST_pMNIST

**Comment by Laborieux-Axel** [comment]
Hi,
Thanks for reaching out! As far as I know this instability issue appeared with later versions of PyTorch and comes from the running stats of the `batchnorm2d`. So your options are either downgrading torch to 1.1.0, which might be quite legacy at this point, or setting `track_running_stats = False` in the `batchnorm2d` functions. Let me know if this helps!

**Comment by chedlyba** [comment]
Hi,

I changed `track_running_stats = False`  and it works perfectly fine now. Thank you for your help.

**Comment by qqish** [comment]
> Hi, Thanks for reaching out! As far as I know this instability issue appeared with later versions of PyTorch and comes from the running stats of the `batchnorm2d`. So your options are either downgrading torch to 1.1.0, which might be quite legacy at this point, or setting `track_running_stats = False` in the `batchnorm2d` functions. Let me know if this helps!

I have set the "track_running_stats=False" in the "batchnorm2d", and got the paper results. But there is another problem when setting that under EWC algorithms. When I run --ewc and --ewc-lambda 5000.0, and set the "track_running_stats=False" , the estimate_fisher function goes wrong. Maybe because the batch size in dset_data_loader is 1? and the error is shown as "except more than 1 value per channel when training, got input size torch.Size([1,4096])" in the line "x=self.layers[self.norm+str(layer+1)](x)". 
So.. what should I do? using the "track_running stats=True" to train the EWC will cause the decay of the first task petformance, but using the "False" one will report the error. Hope that I have explained clearly.
